### PR TITLE
Fixes RuntimeEnvironmentModifier path

### PR DESCRIPTION
**Pull request by nakomis**

### CODE REVIEW

File: brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/spec/RuntimeEnvironmentModifier.java [hunk 1]
```diff
@@ -49,41 +45,28 @@ public RuntimeEnvironmentModifier(ManagementContext mgmt, ToscaFacade<? extends
...
             // Copy all files in resource.
             try {
                 Files.walkFileTree(optionalResourcesRootPath.get(), new SimpleFileVisitor<Path>() {
                     @Override
                     public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IOException {
-                        String tempDest = Os.mergePaths(tempRoot, file.getFileName().toString());
-                        String finalDest = Os.mergePaths(destRoot, file.getFileName().toString());
-                        filesToCopy.put(file.toAbsolutePath().toString(), tempDest);
-                        preInstallCommands.add(String.format("mv %s %s", tempDest, finalDest));
+                        filesToCopy.put(file.toAbsolutePath().toString(), file.getFileName().toString());
```
Comment (sjcorbett) [inline brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/spec/RuntimeEnvironmentModifier.java]: Doesn't this risk overwriting files? e.g. if a project had resources `config/foo` and `config2/foo`. If possible the approach should be to copy the files with their directory structure intact.

**Comment by nakomis** [comment]
PR comments addressed

**Comment by sjcorbett** [comment]
Looks good. Can you also reenable and check `projects/mysql-topology.tosca.yaml` in `AlienSamplesAcceptanceTest`?

**Comment by sjcorbett** [comment]
Will merge this. Would be good to follow up on the last comment about the QA tests.

**Pull request merged**